protoutils/protoutils_test.go

```diff
@@ -332,24 +330,33 @@ func TestStructToStructPb(t *testing.T) {
 	}
 }
 
+func TestStructToStructPbOmitEmpty(t *testing.T) {
```

Comment:
Can you add the same test to TestStructToStructPb and ensure that it instead equals {} there?

Reply:
Spoke with Vijay offline; he didn't end up adding a test for this because one already exists [here](https://github.com/viamrobotics/goutils/pull/204/files#diff-9c3c3101f109f0429d739b33ceb0acd4463bad9e25bc8c8d1b2769d000a7c501R75)

Alexa had commented this elsewhere but I missed it